scripts/generator.ts

```diff
@@ -220,12 +226,12 @@ function addGenerics(node: Node, elementName: string) {
     // const events = {
     //   onActiveItemChanged: "active-item-changed",
     //   ...
-    // } as GridEventMap<unknown>;
+    // } as GridEventMap<any>;
     //                     ^ adding this type argument
     return ts.factory.createTypeReferenceNode(
       ts.factory.createIdentifier(EVENT_MAP),
       isEventMapGeneric
-        ? genericTypeNames.map(() => ts.factory.createKeywordTypeNode(ts.SyntaxKind.UnknownKeyword))
+        ? genericTypeNames.map(() => ts.factory.createKeywordTypeNode(ts.SyntaxKind.AnyKeyword))
```

Comment:
Also needed to change the generic type of the Event Map passed to `createComponent` to `any` since `unknown` doesn't satisfy the constraint:

<img width="1096" alt="Screenshot 2024-08-26 at 16 47 42" src="https://github.com/user-attachments/assets/5aed79d4-8120-49e3-8979-219b8ba69c9d">